_data/sites.json

```diff
@@ -8990,6 +8990,21 @@
         ]
     },
 
+    {
+        "name": "Rituals.com",
+        "url": "https://service.rituals.com/s/contact?topic=a2L4L0000008xwDUAQ&language=de&country=DE",
+        "difficulty": "medium",
+        "notes": "You have to claim a GDPR case for the right to delete your data.",
```

Comment:
Does that mean that you do have to talk to a customer service person anyway?

Reply:
You have to send a message via their contact form https://service.rituals.com/s/contact?topic=a2L4L0000008xwDUAQ&language=de&country=DE to file a GDPR Claim (as there's no button to delete an account anywhere). After the message a staff member from Rituals has to do the actual deletion inside their company administration tools. There's no verbal contact needed (thankfully).

Reply:
I see, then I think its difficulty should be changed to hard.